pandas/io/excel.py

```diff
@@ -25,6 +25,7 @@
 import pandas.core.common as com
 from warnings import warn
 from distutils.version import LooseVersion
+from parsers import _NA_VALUES
```

Comment:
add the full import path. Actually we should move these to ``pandas.io.common``.